international-journal-of-quantum-chemistry.csl

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>International Journal of Quantum Chemistry</title>
+    <id>http://www.zotero.org/styles/international-journal-of-quantum-chemistry</id>
+    <link href="http://www.zotero.org/styles/international-journal-of-quantum-chemistry" rel="self"/>
+    <author>
+      <name>Ji Young Park</name>
+      <email>mutnauq7@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="chemistry"/>
+    <issn>1097-461X</issn>
+    <summary>International Journal of Quantum Chemistry style.</summary>
+    <updated>2015-09-30T01:02:53+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+    <link href="http://onlinelibrary.wiley.com/journal/10.1002/(ISSN)1097-461X"/>
```

Comment:
This should be `<link href="http://onlinelibrary.wiley.com/journal/10.1002/(ISSN)1097-461X" rel="documentation"/>`